airflow/dag_processing/processor.py

```diff
@@ -534,7 +535,9 @@ def manage_slas(cls, dag_folder, dag_id: str, session: Session = NEW_SESSION) ->
                     email_sent = True
                     notification_sent = True
                 except Exception:
-                    Stats.incr("sla_email_notification_failure", tags={"dag_id": dag.dag_id})
+                    Stats.incr(
+                        "sla_email_notification_failure", tags={"dag_id": dag.dag_id, "email": str(emails)}
```

Comment:
I think this `email` tag is going to be unusable, in some cases it's going to contain a long list of emails, and I'm not sure you can query tags by parts of their name ?
Also, a list of emails belongs more in the logs than in metrics imho. What use case do you see for filtering/expanding metrics based on list of emails notified ?

Reply:
I'm actually going to phone-a-friend on that one.  I pulled that directly from @howardyoo 's POC and noticed it was one he added.  Howard, thoughts?  Is there a specific reason you wanted that in there or can/should I drop it?

Reply:
Sorry to ping you again, I saw you replied to other comments but not this one.  Any thoughts on this one, @howardyoo ?

Reply:
Removed email tag in a commit Coming Soon :tm:   Going to resolve this.